tests/__tests__/tsconfig-string.spec.ts

```diff
@@ -52,4 +52,35 @@ describe('get ts config from string', () => {
     });
   });
 
-});
\ No newline at end of file
+  it('should correctly resolve the "extends" directive', () => {
+    const {getTSConfig} = require('../../src/utils');
+    const result = getTSConfig({
+      '__TS_CONFIG__': 'extends-tsconfig.json'
+    });
+
+    expect(result).toEqual ({
+      'target': 2,
+      'module': 1,
+      'moduleResolution': 2,
+      'noEmitOnError': true,
+      'jsx': 2
+    });
+  });
+
+  it('should correctly override any config in the "extends" directive', () => {
+    const {getTSConfig} = require('../../src/utils');
+    const result = getTSConfig({
+      '__TS_CONFIG__': 'extends-with-overrides-tsconfig.json'
+    });
+
+    expect(result).toEqual ({
+      'target': 1,
+      'module': 1,
+      'moduleResolution': 2,
+      'noEmitOnError': true,
+      'jsx': 2,
+      'noImplicitAny': true
+    });
+  });
+
```

Comment:
Wouldn't it be better to use constants from typescript that represent each of these values instead of numbers?